### DISCUSSION TITLE
"Regionally limited calibration" to correct the cursor/pointer offset that is inherent in larger pen displays near their screen borders?

### DISCUSSION THREAD

**Discussion by godfatherjohn** [OpenTabletDriver/OpenTabletDriver#3107] [Plugins]
Larger pen displays (all of them) have a notable cursor/pointer offset which becomes visible when moving close to the display border.  

Here is some more information from some real pen tablet/display professional: https://docs.thesevenpens.com/drawtab/product-info/wacom/wacom-cintiq-pro/7p-notes-wacom-dth271#pen-tracking-accuracy 

My question is, is there a plugin that allows for calibrating the pen for certain, if possible multiple, limited regions instead of the entire active area? If not, what would it involve to build such a plugin. (Perhaps I might be able to work on it in my spare time.)

**Comment by jamesbt365** [comment]
Tablet Calibration plugin in the plugin manager. But OTD does not support having multiple areas on a single tablet, yes that would technically be possible with a plugin, but it would be messy and no such plugin exists.

Not sure what you would mean otherwise calibrating multiple limited regions, and i generally don't understand why regular calibration tools wouldn't suffice if you get them dialed in right and seeing the complications of having multiple calibrated (to different margins) on the same tablet display that would possibly lead to some form of warping or jumping between boundaries?

**Reply by godfatherjohn** [reply]
Thank you for the swift reply. I haven't tried the calibration plugin yet but will do that at some point. I have been using Windows Ink  (in absolute mode) and overall I am happy with how things are working. However, that little bit of cursor/pointer offset near the boarders is singing that I would love to have accounted for. It's not that it impacts my work in any meaningful way. Rather I'm thinking of optimising something to perfection.
